src/timeline.ts

```diff
@@ -164,13 +177,30 @@ class Timeline implements IAppView {
           events.fire(AppConstants.EVENT_DATASET_SELECTED_RIGHT, d.item);
 
           isClicked = 0;
-          console.log ('second Click');
+          console.log('second Click');
         }
 
 
       });
 
-    //Create Bars
+    }
+
+    // Hide and Show timeline (line + circles)
+    $('.btn-primary').on('click', function (e) {
+      var line = svgtimeline.select('line');
```

Comment:
Avoid `var` (if possible) and use `const` (for non changing variables) and `let` (for changing variables)